mmseg/apis/ote/sample/sample.py

```diff
@@ -74,47 +74,47 @@ def main(args):
     logger.info('Load model template')
     model_template = parse_model_template(args.template_file_path)
 
-    hyper_parameters = model_template.hyper_parameters.data
-    set_values_as_default(hyper_parameters)
```

Comment:
Seems it's ok to remove it due to the explicit setting hyperparameters several lines below. But we need to clean up code and remove the function `set_values_as_default` from here: `mmseg/apis/ote/apis/segmentation/config_utils.py`. Do you want to do it independently or should I reopen PR and make changes?

Reply:
If it is the only thing that should be removed, I will do it (remove `set_values_as_default`) right now

Reply:
Nothing more

Reply:
Done